data/companies.json

```
@@ -127,6 +127,22 @@
         "twitter": "https://twitter.com/bajratech",
         "youtube": null
       }
+    },  
+    {
+      "name": "BlackBox Technologies Pvt. Ltd.",
+      "establishedAt": 1540607682000,
+      "location": "Kupondole, Lalitpur",
+      "website": "https://www.bboxtech.com/",
+      "email": "info@bboxtech.com",
+      "phone": "01-5541320",
```

The phone number format isn't correct. 
```suggestion
      "phone": "+977-15541320",
```